frontend/public/index.html

```diff
@@ -4,7 +4,7 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+        <!-- <link rel="icon" type="image/svg+xml" href="favicon.svg" /> -->
```

Comment:
Why this change, @jeswanke?

Reply:
kept coming up in console that it didn't exist but looks like it exists in release so put it back